### ISSUE TITLE
How does ShapeCrawler handle SmartArt?

### ISSUE THREAD

**Issue by pucx** [ShapeCrawler/ShapeCrawler#716]
In the ShapeCrawler library, are there functions or classes available to handle SmartArt, or are there plans to add support functions?

**Comment by ashahabov** [comment]
No, the library doesn't support SmartArt yet, but yes, it is planned.
